netplugin/agent/state_event.go

```diff
@@ -79,6 +138,11 @@ func processInfraNwCreate(netPlugin *plugin.NetPlugin, nwCfg *mastercfg.CfgNetwo
 		return err
 	}
 
+	// add host access routes for vxlan networks
+	if nwCfg.NetworkName == contivVxGWName {
```

Comment:
Basically, this can be used only with a single tenant for now. Host access will be available to the tenant where this interface belongs.